hub/management/commands/import_council_scorecards_score.py

```diff
@@ -0,0 +1,167 @@
+from django.conf import settings
+
+import pandas as pd
+from tqdm import tqdm
+
+from hub.import_utils import filter_authority_type
+from hub.models import DataSet, DataType
+
+from .base_importers import BaseImportFromDataFrameCommand, MultipleAreaTypesMixin
+
+declare_map = {
+    "Y": "Yes",
+    "N": "No",
+}
+
+
+class Command(MultipleAreaTypesMixin, BaseImportFromDataFrameCommand):
+    cons_row = "gss_code"
+    message = "Importing council scorecards data"
+    uses_gss = True
+    do_not_convert = True
+
+    data_file = settings.BASE_DIR / "data" / "2023_scorecards_data.csv"
+
+    area_types = ["STC", "DIS"]
+
+    defaults = {
+        "label": "Council Climate Action Scorecard",
+        "description": "",
+        "data_type": "percent",
+        "category": "place",
+        "release_date": "2023",
+        "source_label": "Data from the Climate Emergency UK.",
+        "source": "https://councilclimatescorecards.uk/",
```

Comment:
I wonder, could we make this a direct link to the council’s page on the scorecards site? At least that way, there’ll be a direct link to more detail from the dataset card on the area page.